src/App.js

```diff
@@ -4,24 +4,28 @@ import './App.css';
 import ToDoForm from './components/ToDoForm/ToDoForm';
 import ToDoList from './components/ToDoList/ToDoList';
 import ToDoControl from './components/ToDoControl/ToDoControl';
+import ToDoModal from './components/ToDoModal/ToDoModal';
 
 // Context section
 import {ToDoContext} from './ContextApi';
 
 // model section
 import ToDoModel from './models/ToDoModel';
 
+// Test commit
+
 function App() {
-  const { active, todos, completed, pending, ...toDoService } = ToDoModel();
+  const { active, todos, completed, pending, modal, ...toDoService } = ToDoModel();
 
   return (
     <div className="App">
-      <ToDoContext.Provider value={{active, todos, completed, pending, toDoService}}>
+      <ToDoContext.Provider value={{active, todos, completed, pending, modal, toDoService}}>
```

Comment:
hi man, "great work". maybe we can sit together and see if there is other approach, i have an idea

Reply:
Updating approach as suggested